Library/Formula/skinny.rb

```diff
@@ -1,5 +1,3 @@
-require "formula"
-
 class UniversalNpm < Requirement
```

Comment:
I'm not completely sure why this is a requirement given we ship Node `--with-npm` by default. IMO we should scrap the block and requirement completely.